snap/container.go

```diff
@@ -33,7 +33,10 @@ import (
 // Container is the interface to interact with the low-level snap files
 type Container interface {
 	// ReadFile returns the content of a single file from the snap.
-	ReadFile(relative string) (content []byte, err error)
+	ReadFile(relative string) ([]byte, error)
+
+	// ReadDir returns the content of a single directory inside the snap.
+	ReadDir(path string) ([]os.FileInfo, error)
```

Comment:
Sure, in my use-case I only need the file name, but changing this to `[]string` limits the amount of information we get out of it, of course. Can you think of something else, or do you just want `[]string`?